main/models/mock/character-sheet.mocks.ts

```diff
@@ -0,0 +1,32 @@
+import { CharacterSheet } from '../Step1/character-sheet';
+import { MockStats1, MockStats2, MockStatsCombat1, MockStatsCombat2 } from './stats.mocks';
+import {
+  MockCharacterClass1,
+  MockCharacterClass2,
+  MockCharacterClassCombat1,
+  MockCharacterClassCombat2,
+} from './character-class.mock';
+import { MockItem1, MockItem2, MockItemCombat1, MockItemCombat2 } from './item.mocks';
+
+export const MockCharacterSheet1: CharacterSheet = new CharacterSheet('Ghita', 1, MockStats1, MockCharacterClass1, [
+  MockItem1,
```

Comment:
add a few more items in your mock (at least one character sheet should have an armor)